app/models/category.rb

```diff
@@ -17,6 +17,7 @@
 #
 class Category < ApplicationRecord
   include Discard::Model
+  attr_accessor :name, :description
```

Comment:
What is the purpose of this line.

Reply:
I think I saw it in a guide but rereading the serializer instructions I see I got confused.
I’m already removing the line from the model.
Thanks!